Backend_Repo/website/template/website/index.html

```diff
@@ -1,52 +1,60 @@
 {% extends "website/base.html" %} {% block main %}{% load staticfiles %}
+{% block links %}
+<script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.18.1/locale/en-gb.js"></script>
```

Comment:
This is a locale file.
Not Needed.
Just use https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.18.1/moment.js instead